config/manager/controller_manager_config.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: controller-runtime.sigs.k8s.io/v1alpha1
```

Comment:
This file is not used anywhere so it was removed since you've opened the PR. Please remove it from the PR